src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8141,6 +8145,179 @@ namespace ConvectionCoefficients {
         }
     }
 
+    void CalcASTMC1340ConvCoeff(EnergyPlusData &state,
+                                int const SurfNum,                  // surface number for which coefficients are being calculated
+                                Real64 const SurfaceTemperature,    // Temperature of surface for evaluation of HcIn
+                                Real64 const ZoneMeanAirTemperature // Mean Air Temperature of Zone
+    )
+    {
+        int ZoneNum = Surface(SurfNum).Zone;
+        Real64 Volume = Zone(ZoneNum).Volume; // Volume of the zone in m3
+        Real64 Vair = std::pow(Volume, OneThird) * CalcZoneSystemACH(state, ZoneNum) / 3600;
+
+        HConvIn(SurfNum) = CalcASTMC1340ConvCoeff(SurfNum, SurfaceTemperature, ZoneMeanAirTemperature, Vair, Surface(SurfNum).Tilt);
+                 
+        // Establish some lower limit to avoid a zero convection coefficient (and potential divide by zero problems)
+        if (HConvIn(SurfNum) < LowHConvLimit) HConvIn(SurfNum) = LowHConvLimit;
```

Comment:
Is this something that was mimicked from other convection functions or is this something specific to this correlation?

Reply:
It was mimicked from[ `CalcASHRAEDetailedIntConvCoeff` function.](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/ConvectionCoefficients.cc#L2514-L2516)